src/js/testem-component.js

```diff
@@ -463,7 +452,7 @@ fluid.defaults("gpii.testem.base", {
             }
         },
         cwd: "@expand:fluid.module.resolvePath({that}.options.cwd)",
-        user_data_dir: "@expand:gpii.testem.resolveFluidModulePathSafely({that}.options.testemDir)",
+        //user_data_dir: "@expand:gpii.testem.resolveFluidModulePathSafely({that}.options.testemDir)",
```

Comment:
Stray comments here, and on priority: first later on

Reply:
Thanks for catching that, fixed.